packages/core/test/unit/plugins/default/shorthandSyntax.test.js

```diff
@@ -0,0 +1,39 @@
+const cheerio = require('cheerio');
+const shorthandSyntaxPlugin = require('../../../../src/plugins/default/markbind-plugin-shorthandSyntax');
+
+test('processNode should convert shorthand syntax to proper Markbind syntax', () => {
+  const [spanNode] = cheerio.parseHTML('<panel>'
+    + '<span heading>Heading</span></panel>', true)[0].children;
+  shorthandSyntaxPlugin.processNode({}, spanNode);
+  expect(spanNode.type).toEqual('tag');
+  expect(spanNode.name).toEqual('span');
+  expect(spanNode.attribs.class).toEqual('card-title');
+  expect(spanNode.attribs.slot).toEqual('header');
+  expect(spanNode.children[0].data).toEqual('Heading');
+  expect(spanNode.children[0].type).toEqual('text');
+},
+);
+
+test('processNode should not convert span node without heading attribute', () => {
+  const [spanNode] = cheerio.parseHTML('<panel>'
+    + '<span>Heading</span></panel>', true)[0].children;
+  const copy = { ...spanNode };
+  shorthandSyntaxPlugin.processNode({}, spanNode);
+  expect(spanNode).toEqual(copy);
+},
+);
+
+test('processNode should not convert improper syntax nodes', () => {
+  const [divSpanNode] = cheerio
+    .parseHTML('<div><span heading>Heading</span></div>', true)[0].children;
```

Comment:
I think it would be good to add short inline comments to explain how are these syntaxes improper, so readers don't have to refer to the function to infer what the proper syntax should be. It also serves as also a form of verification of what improper syntax we are actually testing.

Reply:
Hi, thanks for the comment. I have adjusted the test cases for the `shorthandSyntax` plugin and included in a comment at the top to show the desired outcome. Hope that's clearer.